front/lib/api/assistant/legacy_agent.ts

```diff
@@ -250,6 +250,7 @@ export async function* runLegacyAgent(
           created: event.created,
           configurationId: configuration.sId,
           messageId: agentMessage.sId,
+          runId: event.runId,
```

Comment:
This file disappears today no need to handle

Reply:
Need to keep it for now because of typings